packages/sdk/src/webhooks/client.ts

```diff
@@ -17,42 +36,42 @@ export class LinearWebhookClient {
   /**
    * Creates a webhook handler function that can process Linear webhook requests
    * @returns A webhook handler function with event registration capabilities.
-   * Returns `(request: Request) => Promise<Response>`
+   * Supports both Fetch API `(request: Request) => Promise<Response>` and
+   * Node.js `(request: IncomingMessage, response: ServerResponse) => Promise<void>`
    */
   public createHandler(): LinearWebhookHandler {
     const eventHandlers = new Map<string, LinearWebhookEventHandler<LinearWebhookPayload>[]>();
 
-    const handler = async (request: Request): Promise<Response> => {
+    const handler = async (
+      requestOrMessage: Request | IncomingMessage,
+      response?: ServerResponse
+    ): Promise<Response | void> => {
+      const env = this.getEnv(requestOrMessage, response);
```

Comment:
nit: it seems slightly awkward to name this `env` since there's already `process.env`, and this isn't an "env" object exactly. Maybe we replace these with "adapter" like:

- NormalizedEnv → HttpAdapter
- getEnv → getHttpAdapter
- createFetchEnv → createFetchAdapter
- createNodeEnv → createNodeAdapter
- local var `env` → `adapter`

or otherwise maybe "context" if not "adapter"?